devicetypes/erocm123/inovelli-2-channel-smart-plug.src/inovelli-2-channel-smart-plug.groovy

```diff
@@ -83,7 +83,7 @@ def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicReport cmd, ep = null)
 			def allOff = true
 			childDevices.each {
 				n ->
-					if (n.currentValue("switch")? != "off") allOff = false
+					if (n.currentState("switch") != null && n.currentState("switch").value != "off") allOff = false
```

Comment:
Any reason that `!= null is included here but not in the other place?

Reply:
...coulda sworn i had it...